fire/fire_test.py

```diff
@@ -19,17 +19,17 @@
 import fire
 from fire import test_components as tc
 from fire import trace
+from fire.core_test import BaseTest
 
 import six
 import unittest
 
 
-class FireTest(unittest.TestCase):
-
+class FireTest(BaseTest):
   def testFire(self):
-    fire.Fire(tc.Empty)
-    fire.Fire(tc.OldStyleEmpty)
-    fire.Fire(tc.WithInit)
+    fire.Fire(tc.Empty, '')
```

Comment:
It sounds like it was happening because sys.argv was being used. Previously it would just return None in the case of an error, but now it's SystemExiting.

Let's (at least for one test) mock argv rather than supplying a command string, so that the common case of no-command is represented in the tests.